MSAL/src/MSALPublicClientApplication.m

```diff
@@ -81,6 +83,15 @@ @interface MSALPublicClientApplication()
 
 @implementation MSALPublicClientApplication
 
++ (void)initialize
+{
+#if TARGET_OS_IPHONE
+    s_defaultKeychainGroup = MSIDKeychainTokenCache.defaultKeychainGroup;
```

Comment:
why not use MSIDKeychainTokenCache.defaultKeychainGroup directly?

Reply:
We have overloaded initializers, and I didn't want to do if target_os_iphone for all of them.

Reply:
Discussed offline that we can make it a method or function, no need for a s_defaultKeychainGroup